### PR TITLE
Docs update 2

### PR DESCRIPTION
**Pull request by zhostasa**
* Three benchmarks renamed
* Created documents for Reporters
* Created separate tab for example configurations
* Removed benchmark-periodic.xml for being redundant
* benchmark-continuous-query.xml needed to be fixed as per experimental stages removal
* ISPN 9 extension query.xml fixed, does not like when default cache does not exist

I did my best to comment on separate benchmarks somewhat, feel absolutely free to advise if you find any of them is deserving of more information (or if I got it wrong)

### CODE REVIEW

File: docs/benchmark_configuration/reporter_csv.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+---
+
+CSV reporter
+------------
+
+CSV reporter is used to store benchmark data into .csv files, usually fot further processing.
```
Comment (jmarkos) [inline docs/benchmark_configuration/reporter_csv.md]: typo: for

File: docs/getting_started/example_configurations.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+**[benchmark-coherence-hazelcast.xml](https://github.com/radargun/radargun/blob/master/extensions/cache/src/main/resources/benchmark-coherence-hazelcast.xml)** - Shows how to run comparative analysis of three products (Coherence, Hazelcast and Infinispan)
+
+**[benchmark-combined-report.xml](https://github.com/radargun/radargun/blob/master/extensions/cache/src/main/resources/benchmark-combined-report.xml)** - Shows how to property handle combining different tests
+
+**[benchmark-condition.xml](https://github.com/radargun/radargun/blob/master/extensions/cache/src/main/resources/benchmark-condition.xml)** - Shows how to use conditions in the `repeat` stage
+
+**[benchmark-continuous-query.xml](https://github.com/radargun/radargun/blob/master/extensions/query/src/main/resources/benchmark-continuous-query.xml)** - Shows how to call continuous queries using `continuous-query` stage
+
+**[benchmark-dist-custom-vars.xml](https://github.com/radargun/radargun/blob/master/extensions/cache/src/main/resources/benchmark-dist-custom-vars.xml)** - Shows how to defin custom properties trough `define` stage
```
Comment (jmarkos) [inline docs/getting_started/example_configurations.md]: typo: define, through

File: docs/getting_started/example_configurations.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+**[benchmark-hazelcast-server.xml](https://github.com/radargun/radargun/blob/master/extensions/cache/src/main/resources/benchmark-hazelcast-server.xml)** - Shows performance test for Hazelcast client/server configuration
+
+**[benchmark-hotrod-rest-memcached.xml](https://github.com/radargun/radargun/blob/master/extensions/cache/src/main/resources/benchmark-hotrod-rest-memcached.xml)** - Shows performance testing for same-server/different-client configurations
+
+**[benchmark-hotrod-spark-mapreduce.xml](https://github.com/radargun/radargun/blob/master/extensions/mapreduce/src/main/resources/benchmark-hotrod-spark-mapreduce.xml)** - Shows how to benchmark MapReduce operations on Infinispan coupled with Spark
+
+**[benchmark-hotrod-streaming.xml](https://github.com/radargun/radargun/blob/master/extensions/cache/src/main/resources/benchmark-hotrod-streaming.xml)** - Shows how to compare memory usage on Infinispan clients using and not using Streaming API
+
+**[benchmark-iteration.xml](https://github.com/radargun/radargun/blob/master/extensions/cache/src/main/resources/benchmark-iteration.xml)** - Shows how to used`iterate` stage to iterate over all entries in the cache
```
Comment (jmarkos) [inline docs/getting_started/example_configurations.md]: typo: use

**Comment by mgencur** [comment]
@jmarkos , are you going to integrated it?

**Comment by jmarkos** [comment]
@mgencur sure if you don't have any comments (since @zhostasa specifically requested a review from you - not sure if it had any reason)

**Comment by mgencur** [comment]
@jmarkos , it's up to you;-)

**Comment by zhostasa** [comment]
Please do remember I have some fixes to do:-)
Also, @mgencur is the requester of the changes so it is up to him to decide if these fulfil his request, but other point of view is more then welcome

**Comment by zhostasa** [comment]
All fixed

**Comment by zhostasa** [comment]
Someone is dragging his feet so I added syntax highlighting to the code blocks so I have cause to say something and get him to merge this. Also - pretty-pretty *wink*

**Comment by jmarkos** [comment]
Merged, thanks. I knew you were holding something back, so I waited a bit.

**Pull request merged**